airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java

```diff
@@ -39,7 +43,20 @@ public JsonNode toDatabaseConfig(final JsonNode redshiftConfig) {
             redshiftConfig.get("host").asText(),
             redshiftConfig.get("port").asText(),
             redshiftConfig.get("database").asText()));
+
+    if (redshiftConfig.has("schemas") && redshiftConfig.get("schemas").isArray()) {
```

Comment:
we could make "schemas" string constant here

Reply:
Done, thanks